Packages/com.unity.inputsystem/InputSystem/Unity.InputSystem.asmdef

```diff
@@ -2,7 +2,8 @@
     "name": "Unity.InputSystem",
     "rootNamespace": "",
     "references": [
-        "Unity.ugui"
+        "Unity.ugui",
+        "Unity.Profiling.Core"
```

Comment:
Wondering if this might create some overhead? Probably we could set a define like `INPUTSYSTEM_USE_CUSTOMPROFILER` or something similar, in case users don't want to have this code in their release builds.